script/sort.py

```diff
@@ -0,0 +1,9 @@
+import argparse
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument('fruit', nargs='*')
+args = parser.parse_args()
+sortList = sorted(args.fruit)
+for i in sortList:
+  print(i, end=' ')
```

Comment:
This won't print in desired format. It is nice that you wrote correct for loop, but this can written using join() function:
https://www.w3schools.com/python/ref_string_join.asp

Also as a second tip (Not needed here, but very useful in general):
If you want to print variable with some text like:
```python
print("Hello ", name)
```
It is better to use string interpolation
```python
print("Hello {}".format(name))
```
More can be found here:
https://www.programiz.com/python-programming/string-interpolation

Reply:
I found something called syntax to unpack arguments and it works.

Reply:
Yes it does. But you won't usually find something like this in python.
The second solution is much more better. :-)

Anyway: good Job on this task!